scripts/openvpn/makeOVPN.sh

```diff
@@ -405,6 +405,26 @@ else
 
 fi
 
+# Find an unused number for the last octet of the client IP
+for i in {2..254}; do
+    # find returns 0 if the folder is empty, so we create the 'ls -A [...]'
+    # exception to stop at the first static IP (10.8.0.2). Otherwise it would
+    # cycle to the end without finding and available octet.
+    if [ -z "$(ls -A /etc/openvpn/ccd)" ] || ! find /etc/openvpn/ccd -type f -exec grep -q "10.8.0.$i" {} +; then
```

Comment:
Greping for 10.8.0.$i doesn't seem right .. there might be cases where users change the vpn IP ranges ... instead should  pickup from the setup vars and then do the rest of the magic